config.py

```diff
@@ -74,6 +74,15 @@ class Config(object):
         'g-cloud-5': 'g-cloud-6',
     }
 
+    DM_FEEDBACK_FORM = {
```

Comment:
Assuming the feedback form is available across the site, do we have a shared config place we could put this?

Reply:
AFAIK, we don't have any config that's shared between the apps.

But in this case only the buyer app will need that particular config, because there's only one submission endpoint, and it's here. Only the template will end up being shared.

Reply:
Oh of course. Cool.